lib/types/models.ts

```diff
@@ -380,6 +381,7 @@ export interface DeviceType {
 	name: string;
 	is_private: boolean;
 	logo: string | null;
+	contract: Contract | null;
```

Comment:
I also just realized that the types of the image.contract & release.contract are `string | null`, which I guess are wrong...
Would be nice to fix them in a separate patch release.

Reply:
Yeah, I noticed this. I'll fix these in a separate patch as well